plugin/storage/integration/elasticsearch_test.go

```diff
@@ -168,6 +168,7 @@ func (s *ESStorageIntegration) initSpanstore(allTagsAsFields, archive bool) erro
 		IndexPrefix:       indexPrefix,
 		MaxSpanAge:        maxSpanAge,
 		TagDotReplacement: tagKeyDeDotChar,
+		AllTagsAsFields:   allTagsAsFields,
```

Comment:
can there be a test that validates this new behavior?

Reply:
The integration tests were strong and gave the expected behaviour. I can add the unit tests.

Reply:
but those tests didn't catch the issue reported by user?